tests/dragonfly/memory_test.py

```diff
@@ -24,10 +24,12 @@
 # We limit to 5gb just in case to sanity check the gh runner. Otherwise, if we ask for too much
 # memory it might force the gh runner to run out of memory (since OOM killer might not even
 # get a chance to run).
-@dfly_args({"proactor_threads": 4, "maxmemory": "5gb"})
+@dfly_args({"proactor_threads": 2, "maxmemory": "5gb"})
 async def test_rss_used_mem_gap(df_factory, type, keys, val_size, elements):
     dbfilename = f"dump_{tmp_file_name()}"
-    instance = df_factory.create(dbfilename=dbfilename)
+    instance = df_factory.create(
+        dbfilename=dbfilename, compression_mode=0, serialization_max_chunk_size=8192, num_shards=2
```

Comment:
made a huge difference -- especially locally :shrug:

Reply:
sure but I find it weird that we split args between `df_factory.create` and `dfly_args`. Is this necessary in this test?